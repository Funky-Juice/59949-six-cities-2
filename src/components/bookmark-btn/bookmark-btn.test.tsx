import * as renderer from 'react-test-renderer';
import BookmarkBtn from './bookmark-btn';
import offerTestObj from '../../mocks/test-offer';

it(`BookmarkBtn correctly renders`, () => {
  const CB = jest.fn();
  const tree = renderer
    .create(<BookmarkBtn
      offer={offerTestObj}
      btnClass={`someClassName`}
      isVisible={true}
      deleteOffer={CB}
      setBookmark={CB}
      onVisibleChange={CB}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
