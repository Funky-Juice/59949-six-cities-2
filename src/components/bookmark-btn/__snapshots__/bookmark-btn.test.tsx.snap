// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkBtn correctly renders 1`] = `
<button
  className="
          button
          someClassName__bookmark-button
          someClassName__bookmark-button--active
        "
  onClick={[Function]}
  type="button"
>
  <svg
    className="someClassName__bookmark-icon"
    height="19"
    width="18"
  >
    <use
      xlinkHref="#icon-bookmark"
    />
  </svg>
  <span
    className="visually-hidden"
  >
    In bookmarks
  </span>
</button>
`;
